builtin/tag.c

```diff
@@ -106,7 +106,28 @@ static int delete_tag(const char *name, const char *ref,
 static int verify_tag(const char *name, const char *ref,
 				const unsigned char *sha1)
 {
-	return gpg_verify_tag(sha1, name, GPG_VERIFY_VERBOSE);
+	int flags;
+	flags = GPG_VERIFY_VERBOSE;
+
+	if (fmt_pretty) {
+		//verify_ref_format(fmt_pretty);
+
+		// Create a `ref_array_item` in order to use `show_ref_array_item`
+		// XXX: Just copies code from `static new_ref_array_item` in ref_filter.c
+		struct ref_array_item *ref_item;
+		FLEX_ALLOC_STR(ref_item, refname, name);
+		hashcpy(ref_item->objectname, sha1);
+		ref_item->kind = FILTER_REFS_TAGS;
+		ref_item->flag = 0;
+
+		show_ref_array_item(ref_item, fmt_pretty, 0);
+		free((char *)ref_item->symref);
+		free(ref_item);
+
+		flags = GPG_VERIFY_QUIET;
```

Comment:
Not with this patch. Now it shows either the `GPG_VERIFY_VERBOSE` output like before or with a given `--format string`, the formatted tag object.

I see (at least) two ways of adding this functionality though:
 - Toggle `GPG_VERIFY_QUIET` with another command line argument, e.g. 
`git tag -v --quiet <tagname>...`, independently of whether a format string is provided or not. Which in case one is provided would stack both outputs above each other (which is a little clumsy). 

 - Somehow make the GPG output available to  [`show_ref_array_item`](https://github.com/lukpueh/git/blob/master/ref-filter.c#L1601-L1638), which now handles the formatting of the `tag` object, in order to enable such calls like:
`git tag -v --format="%(tag)%(gpgresult)%(date)" <tagname>...` .

Although the latter is neater from a user's point of view, it would require massive code changes. In this case I would even take out the formatting from [`ref-filter.c`](https://github.com/lukpueh/git/blob/master/ref-filter.c) because ref filtering and ref signature verification really should not be coupled.